Formula/otc-auth.rb

```diff
@@ -0,0 +1,26 @@
+# Adds the otc-auth project to homebrew
+class OtcAuth < Formula
+  desc "Open Source CLI for the Open Telekom Cloud written in go"
+  homepage "https://github.com/iits-consulting/otc-auth"
+
+  url "https://github.com/iits-consulting/otc-auth.git",
+      tag:      "v2.0.0",
+      revision: "86b76b04813ce94cfaacd95f8653f2fe13851a60"
+  license "MIT"
```

Comment:
Just make it GPLv3 and we are good to go!
https://github.com/iits-consulting/otc-auth/blob/main/LICENSE